maintainer/cuda_build.sh

```diff
@@ -0,0 +1,31 @@
+GIT_COMMIT=$(git rev-parse HEAD)
+URL=$(echo "https://gitlab.icp.uni-stuttgart.de/kai/espresso-build/-/jobs/${CI_JOB_ID}")
```

Comment:
Create a public namespace "espressomd" on our Gitlab server and make a new public project "espresso" there. Use that instead.